src/main/java/com/android/tools/build/bundletool/io/ApkSerializerManager.java

```diff
@@ -80,17 +83,21 @@
   private final AppBundle appBundle;
   private final ApkSetBuilder apkSetBuilder;
 
+  private final boolean verbose;
+
   public ApkSerializerManager(
       AppBundle appBundle,
       ApkSetBuilder apkSetBuilder,
       ListeningExecutorService executorService,
       ApkListener apkListener,
       ApkModifier apkModifier,
+      boolean verbose,
       int firstVariantNumber) {
     this.appBundle = appBundle;
     this.apkSetBuilder = apkSetBuilder;
     this.apkPathManager = new ApkPathManager();
     this.executorService = executorService;
+    this.verbose = verbose;
```

Comment:
nit: Move this line two below to keep the same order as the constructor parameters (and ideally the same order as how the fields are defined in the class -- although I realize they're already not in order).